packages/components/src/styles/tile/button-link-tile-styles.ts

```diff
@@ -44,6 +47,9 @@ export const getButtonLinkTileStyles = (
               fontWeight: getFontWeight(w === 'semibold' ? 'semi-bold' : w), // mapping of the deprecated weight semibold
             }))
           ),
+          ...buildResponsiveStyles(background, (b: Theme) => ({
+            color: getThemedColors(isThemeDark(b) ? 'dark' : 'light').primaryColor,
```

Comment:
Wouldn't this have the same effect?
```suggestion
            color: getThemedColors(b).primaryColor,
```